docs/writing-rules/rule-syntax.md

```diff
@@ -81,6 +81,17 @@ The `patterns` operator performs a logical AND operation on one or more child pa
 
 <iframe src="https://semgrep.dev/embed/editor?snippet=Q83q" border="0" frameBorder="0" width="100%" height="432"></iframe>
 
+### Evaluation strategy
```

Comment:
```suggestion
### Semgrep rule evaluation strategy
```

Reply:
WIP title. Open to suggestions.

Reply:
It's already nested under `patterns` so I'm not sure we need the extra long title here. It also sounds a bit strange to me to say "evaluation strategy of patterns", wouldn't it be "for" rather than "of", and we should use `patterns` rather than just "patterns", because it refers to the `patterns` operator in particular.

Reply:
What is funnel? Also not sure about the word "strongest".